index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="Author" content="Shubh M Das">
+    <link rel="stylesheet" href="style.css">
+    <title>portfolio</title>
+</head>
+
+<body>
+   <header>
+        <div class="container">
+            <div id="branding">
+                <a href="#">My Logo</a>
```

Comment:
- [ OPTIONAL] Great job adding your logo into the `a`  tag 👍🏻  but I think it will be better if  that can have a link to the home page.
Eg:  
````
 <a href="./index.html">My Logo</a>